docker/compose/dev/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '3.8'
+
+services:
+  ssi-agent:
+    container_name: ssi-agent
+    entrypoint: /bin/sh
+    command: -c "while true; do sleep 30; done"
+    environment:
+      - DEMO_HOST_ADDRESS="https://ssi.sphereon.com/demo"
```

Comment:
This should be added by the user and explained in the documentation. Otherwise everyone will fire up the docker compose with a host address they can never have

Reply:
removed and added description in the readme